### DISCUSSION TITLE
CDN for JointJS 4.0.x?

### DISCUSSION THREAD

**Discussion by dmb0058** [clientIO/joint#2549] [Q&A] [answered]
### Introduction

Just wondering if there's a version of JointJS 4.0.x available via cdn?
The examples include "https://cdnjs.cloudflare.com/ajax/libs/@joint/core/4.0.0/joint.js" but this doesn't seem to exist.

David

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by jamesgeorgewilliams** [comment, marked as answer]
We are still waiting for `cdnjs` to add the `@joint/core` package, but the following providers have it.

https://www.jsdelivr.com/package/npm/@joint/core
https://www.jsdelivr.com/package/npm/@joint/layout-directed-graph

https://unpkg.com/@joint/core@4.0.1/dist/joint.min.js
https://unpkg.com/@joint/layout-directed-graph@4.0.1/dist/DirectedGraph.min.js
